src/Resources/config/slot.yml

```diff
@@ -255,3 +255,8 @@ services:
         class: EzSystems\PlatformHttpCacheBundle\SignalSlot\SectionService\UpdateSectionSlot
         parent: ezplatform.http_cache.signalslot.abstract
         tags: [{ name: ezpublish.api.slot, signal: SectionService\UpdateSectionSignal }]
+
+    ezplatform.http_cache.signalslot.assign_section_to_subtree:
+        class: EzSystems\PlatformHttpCacheBundle\SignalSlot\SectionService\AssignSectionToSubtreeSlot
+        parent: ezplatform.http_cache.signalslot.abstract
+        tags: [{ name: ezpublish.api.slot, signal: SectionService\AssignSectionToSubtreeSignal}]
```

Comment:
cs :)

Reply:
Done in https://github.com/ezsystems/ezplatform-http-cache/pull/106/commits/1b208653bd4c3f53409440a7c2037d19d4f5ebaf, thanks!